### DISCUSSION TITLE
Style : when option list is open

### DISCUSSION THREAD

**Discussion by imdkbj** [MobileReality/react-native-select-pro#206] [Ideas]
**Is your feature request related to a problem? Please describe.**
Style object when option list is open.

**Describe the solution you'd like**
activeStyle:{
borderColor:'green'
}

**Describe alternatives you've considered**
use of onSelectOpened and onSelectClosed to toggle the color in local state

**Comment by kkaplinski** [comment]
@imdkbj, Thanks for a suggestion! All of the currently available styling properties you can find here: https://mobilereality.github.io/react-native-select-pro/docs/api/select#styles. I'm moving this thread to discussions for now.
